library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -792,7 +810,12 @@ private void dropPointOnMap(@NonNull LatLng latLng, @Nullable MarkerOptions mark
             markerOptions.setPosition(latLng);
         }
 
-        mapboxMap.addMarker(markerOptions);
+        if (mapboxMap != null) {
```

Comment:
This change is no longer needed, it can be reverted to the previous implementation.